spaceflights/{{ cookiecutter.repo_name }}/src/requirements.txt

```diff
@@ -6,6 +6,7 @@ jupyter~=1.0
 jupyter_client>=5.1, <7.0
 jupyterlab~=3.0
 kedro[pandas.CSVDataSet,pandas.ExcelDataSet]=={{ cookiecutter.kedro_version }}
+openpyxl==3.0.9
```

Comment:
Can we relax this a bit? Develop has `openpyxl>=3.0.6, <4.0` for `pandas.ExcelDataSet`.

Reply:
Done!